### ISSUE TITLE
No more tracking (0.6.2)

### ISSUE THREAD

**Issue by spam8117** [stevschmid/track-o-bot#120]
Since yesterday Track-O-Bot is no longer tracking my games (Ranked, Arena...).
=> see screenshot, last recorded games are from Wednesday :/

[08:08:18] INFO: Hearthstone window found
[09:25:45] INFO: Hearthstone window was closed

**Comment by stevschmid** [comment]
Are you on Mac or Windows?

**Comment by spam8117** [comment]
Hi,
I am on Windows (10). I tried to reinstall Track-O-Bot, then closed Hearthstone, opened Track-O-Bot first, then Hearthstone again and now it works again.
Are there any issues related to this? Because usually Track-O-Bot starts with Windows, so it is on before I start Hearthstone anyways.

I also found another solution, does this make any sense to you:
https://www.reddit.com/r/trackobot/comments/3vz4at/not_tracking_games

Thanks :)
